src/frontend/apps/e2e/__tests__/app-desk/header.spec.ts

```diff
@@ -35,21 +35,17 @@ test.describe('Header', () => {
     await expect(header.getByText('My account')).toBeVisible();
   });
 
-  test('checks logout button', async ({ page }) => {
+  test('checks logout button is visible', async ({ page }) => {
```

Comment:
I would suggest adding back this test when the e2e will run in a K8s setup using Agent Connect integration environment as OIDC provider. I can open an issue.

Reply:
:ok_hand: